main.go

```diff
@@ -124,7 +125,13 @@ func decryptSymmetric(keyName string, ciphertext []byte) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	return resp.Plaintext, nil
+
+	decoded, err := b64.StdEncoding.DecodeString(string(resp.Plaintext))
+	if err != nil {
+		return resp.Plaintext, nil
```

Comment:
If we're not able to basee64 decode, are we fine just returning the b64 encoded string? it kinda feels like we'd want to propagate the error up, but maybe theres a reason not to. at the very least, it seems like we'd probably want some debug logging or something here saying that it failed

Reply:
good point, but this is the backwards compatibility layer. Older sctl files wont have the double base64 decoding so its presumed that its a pre v0.7.0 secret and it just gets returned. it shouldn't be a base64 encoded string here, but i suppose there may be a situation where that occurs. 

i need to figure out how to stub out kms so i can write test cases for this :/

Reply:
@edaniszewski I went ahead and patched this with some debug logging, and comments for code clarity/intent.